explainaboard/metric.py

```diff
@@ -218,6 +221,8 @@ def default_name(cls) -> str:
 
     def __init__(
         self,
+        language="en",
```

Comment:
I think these two lines should probably be removed, as it's essentially defining the defaults in two places, which might cause problems later if you change the defaults somewhere.

Reply:
It will lead to an error if these two are removed due to: https://github.com/neulab/ExplainaBoard/blob/35d7af9b858ea7901038333f40581728a9f49e29/explainaboard/processors/processor.py#L142

Reply:
OK, we can fix this in a separate PR when we introduce MetricConfig.